app/main.py

```diff
@@ -1,6 +1,36 @@
+import requests
+import os
+from dotenv import load_dotenv
+
+load_dotenv()
+
+BASE_URL = "https://api.weatherapi.com/v1/current.json"
+CITY_FILTERING = "Paris"
+API_KEY = os.environ.get("API_KEY")
+
+
 def get_weather() -> None:
-    # write your code here
-    pass
+    payload = {"key": API_KEY, "q": CITY_FILTERING}
+    res = requests.get(BASE_URL, params=payload)
+
+    if res.status_code == 200:
+        weather_dict = res.json()
+
+        location = weather_dict.get("location").get("name")
```

Comment:
Utilizing two .get in a string is not advisable, especially without a default argument, in this case  dictionary.
Fix in all places